packages/camera/camera/android/src/test/java/io/flutter/plugins/camera/DartMessengerTest.java

```diff
@@ -54,55 +47,41 @@ public void setUp() {
   }
 
   @Test
-  public void setsStreamHandler() {
-    assertNotNull(fakeBinaryMessenger.getMessageHandler());
-  }
-
-  @Test
-  public void send_handlesNullEventSinks() {
-    dartMessenger.send(DartMessenger.EventType.ERROR, "error description");
+  public void sendCameraErrorEvent_includesErrorDescriptions() {
+    dartMessenger.sendCameraErrorEvent("error description");
 
     List<ByteBuffer> sentMessages = fakeBinaryMessenger.getMessages();
-    assertEquals(0, sentMessages.size());
+    assertEquals(1, sentMessages.size());
+    MethodCall call = decodeSentMessage(sentMessages.get(0));
+    assertEquals(DartMessenger.EventType.ERROR.toString().toLowerCase(), call.method);
```

Comment:
nit: We typically prefer using literals in tests over constants. It makes it easier to read and self contained. (e.g. "error" instead of `DartMessenger.EventType.ERROR.toString().toLowerCase()`). Up to you though.